kubernetes/apps/networking/cloudflared/ks.yaml

```diff
@@ -0,0 +1,22 @@
+---
+apiVersion: kustomize.toolkit.fluxcd.io/v1
+kind: Kustomization
+metadata:
+  name: cluster-apps-cloudflared
+  namespace: flux-system
+spec:
+  dependsOn:
+    - name: cluster-apps-ingress-nginx
```

Comment:
I'd be fine omitting this since both can come online at the same time or in any order, what do you thin?